code/_helpers/mobs.dm

```diff
@@ -49,8 +49,7 @@ proc/random_name(gender, species)
 		var/datum/species/current_species = get_species_by_key(species)
 		if(current_species)
 			var/decl/cultural_info/current_culture = SSlore.get_culture(current_species.default_cultural_info[TAG_CULTURE])
-			if(current_culture)
-				return current_culture.get_random_name(null, gender)
+			return current_culture?.get_random_name(null, gender)
```

Comment:
Non-equivalent control flow.
Before:
* If current_culture != null then return current_culture.get_random_name(null, gender)
* Otherwise, return some-random name
Now
* Return current culture or null

Reply:
Ah shoot. Didn't mean to keep this change